src/event.rs

```diff
@@ -1068,6 +1068,23 @@ pub enum MouseScrollDelta {
     PixelDelta(PhysicalPosition<f64>),
 }
 
+/// The request to change the inner size synchronously was ignored.
+///
+/// See [`InnerSizeWriter::request_inner_size`] for details.
+#[derive(Debug)] // Explicitly not other traits, in case we want to extend it in the future
+#[non_exhaustive]
+pub struct RequestIgnored {
+    _priv: (),
+}
```

Comment:
Why do we need `non_exhaustive` if it has a private field?

Reply:
Oh yeah, we don't. I'll remove that